display/displaytest/text.go

```diff
@@ -0,0 +1,207 @@
+// Copyright 2024 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package displaytest
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"periph.io/x/conn/v3/display"
+)
+
+// TestTextDisplay exercises the methods provided by the interface. It can be
+// used interactively as a quick smoke test of an implementation, and from test
+// routines. This doesn't test brightness or contrast to avoid EEPROM wear
+// issues.
+func TestTextDisplay(dev display.TextDisplay, interactive bool) []error {
+	result := make([]error, 0)
```

Comment:
`var result []error` will save an unnecessary memory allocation

Reply:
As you wish.